### PR TITLE
Kehaar configured

### PR DESCRIPTION
**Pull request by tie-rack**
Another `kehaar.configured/init!` PR.

### CODE REVIEW

File: resources/config.edn [hunk 1]
```diff
@@ -1,36 +1,135 @@
...
+    {:queue "electorate-works.election.election-users"
+     :channel election-notification-works.channels/electorate-election-users
+     :response true
+     :timeout 30000}
+    {:queue "election-works.election.read"
+     :channel election-notification-works.channels/election-read
+     :response true
+     :timeout 30000}
+    {:queue "election-notification-works.voter-guide.notify"
+     :channel election-notification-works.channels/voter-guide-notify-ext}]
```
Comment (danielglauser) [inline resources/config.edn]: Missing `:timeout 3000`. Needle in a haystack right there.

File: resources/config.edn [hunk 2]
```diff
@@ -1,36 +1,135 @@
...
-                     "election-works.election.read" {:exclusive false :durable true :auto-delete false}}
-            :topics {"events" {:durable true :auto-delete false}}}
+{:rabbitmq
+ {:connection {:host #resource-config/env "RABBITMQ_PORT_5672_TCP_ADDR"
+               :port #resource-config/edn #resource-config/env "RABBITMQ_PORT_5672_TCP_PORT"}
+  :kehaar
+  {:incoming-services
+   [{:queue "election-notification-works.ok"
+     :f election-notification-works.handlers/ok
+     :response true}
```
Comment (danielglauser) [inline resources/config.edn]: Missing `:prefetch-limit 2`?

File: resources/config.edn [hunk 3]
```diff
@@ -1,36 +1,135 @@
...
+     :prefetch-limit 10}]
+
+   :external-services
+   [{:queue "notification"
+     :channel election-notification-works.channels/notification-send}
+    {:queue "notification-works.notification.schedule"
+     :channel election-notification-works.channels/notification-schedule}
+    {:queue "notification-works.notification.unschedule"
+     :channel election-notification-works.channels/notification-unschedule}
+    {:queue "notification-works.notification.search"
```
Comment (danielglauser) [inline resources/config.edn]: Although there weren't `:timeout`s on all of these would it make sense to add them now? What happens if no timeout is specified?

File: resources/config.edn (hunk 1 above)
Comment (tie-rack) [inline resources/config.edn]: That's a fire-and-forget service (`:response` isn't set), thus a timeout doesn't make sense.

File: resources/config.edn (hunk 3 above)
Comment (tie-rack) [inline resources/config.edn]: The default timeout is 1000ms.... but you're right, these `:response true` services had timeouts before. Adding them back.

File: resources/config.edn (hunk 2 above)
Comment (tie-rack) [inline resources/config.edn]: Good point. I'll limit it to 2 threads as well.

File: resources/config.edn (hunk 1 above)
Comment (danielglauser) [inline resources/config.edn]: Nice, I missed that.

**Pull request merged**